src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer starts working");

```

Comment:
The message 'Bulldozer starts working' could be more informative. According to the checklist, it should indicate what type of `Machine` is working. Consider including additional details that describe the work being done by the `Bulldozer`.